lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -62,8 +82,54 @@ public List<Feature> getInstalledFeatureList() {
         return this.installedFeatureList;
     }
 
-    public void setInstalledFeatureList(List<Feature> installedFeatureList){
+    public void setInstalledFeatureList(List<Feature> installedFeatureList) {
         this.installedFeatureList = installedFeatureList;
     }
 
+    public void initConfigFileList() {
+        try {
+            List<Path> configPathsList = Files.find(Paths.get(getWorkspaceURI()), Integer.MAX_VALUE, (filePath, fileAttributes) -> 
+                    LibertyUtils.isServerXMLFile(filePath.toString()) || (fileAttributes.isDirectory() && filePath.toString().endsWith("configDropins")))
+                    .collect(Collectors.toList());
+            for (Path configPath : configPathsList) {
+                if (LibertyUtils.isServerXMLFile(configPath.toString())) {
+                    scanXMLforInclude(configPath);
+                } else {
+                    Files.list(configPath).forEach(path -> configFiles.add(path.toString()));
+                }
+            }
+        } catch (IOException e) {
+            // workspace URI does not exist
+            e.printStackTrace();
+        }
+    }
+
+    public void scanXMLforInclude(Path filePath) {
+        try {
+            String content = new String(Files.readAllBytes(filePath));
+            Matcher m = Pattern.compile("<include.+location=[\"\'](.+)[\"\']/>").matcher(content);
```

Comment:
Is this the best way to find `<include>` elements? When using a regex you need to worry about elements that span multiple lines. We could load a DOM Document and find the include elements. If you think regex is the best way to go, double check it and make sure it will handle multiple lines.